### PR TITLE
Not measure already replicated records twice to avoid ruin the metrics

### PR DESCRIPTION
**Pull request by olegkovalenko**

### CODE REVIEW

File: journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedKeyJournal.scala [hunk 1]
```diff
@@ -18,19 +18,19 @@ trait ReplicatedKeyJournal[F[_]] {
     timestamp: Instant,
     expireAfter: Option[ExpireAfter],
     events: Nel[EventRecord]
-  ): F[Unit]
+  ): F[Boolean]
```
Comment (t3hnar) [inline journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedKeyJournal.scala:21]: can we replace `Boolean` everywhere with type alias `Updated` or `Changed` ?

File: journal/src/test/scala/com/evolutiongaming/kafka/journal/eventual/EventualJournalSpec.scala [hunk 2]
```diff
@@ -507,11 +507,11 @@ object EventualJournalSpec {
 
         def append(partitionOffset: PartitionOffset, events: Nel[EventRecord]) = {
           // TODO expiry: define expireAfter and test
-          journal.append(key, partitionOffset, timestamp, none, events)
+          journal.append(key, partitionOffset, timestamp, none, events).map(_ => ())
```
Comment (t3hnar) [inline journal/src/test/scala/com/evolutiongaming/kafka/journal/eventual/EventualJournalSpec.scala:510]: .void :)

File: journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedKeyJournal.scala (hunk 1 above)
Comment (olegkovalenko) [reply journal/src/main/scala/com/evolutiongaming/kafka/journal/eventual/ReplicatedKeyJournal.scala]: good point

File: journal/src/test/scala/com/evolutiongaming/kafka/journal/eventual/EventualJournalSpec.scala (hunk 2 above)
Comment (olegkovalenko) [reply journal/src/test/scala/com/evolutiongaming/kafka/journal/eventual/EventualJournalSpec.scala]: nice, didn't know about this one :)

**Comment by olegkovalenko** [comment]
Thank you!

**Pull request merged**